.allowed-scripts.mjs

```diff
@@ -0,0 +1,12 @@
+import { configureAllowedScripts } from '@ministryofjustice/hmpps-npm-script-allowlist'
+
+export default configureAllowedScripts({
+  allowlist: {
+    'node_modules/@parcel/watcher@2.5.1': 'ALLOW',
```

Comment:
Is there any way that each line can be accompanied by a comment as to why it is allowed?
I've only heard of two of these packages and don't know why we are allowing the rest 🙂

Reply:
very good idea